test-infra/airflow/Dockerfile

```diff
@@ -0,0 +1,147 @@
+# Docker image for Airflow.
+# This was based on https://github.com/puckel/docker-airflow.
+# However, we modified it because we wanted to use Python2.7 since some
+# of our client libraries require it.
+
+FROM python:2.7-slim
+MAINTAINER Jeremy Lewi
+
+# Never prompts the user for choices on installation/configuration of packages
+ENV DEBIAN_FRONTEND noninteractive
+ENV TERM linux
+
+# Airflow
+ARG AIRFLOW_VERSION=1.8.2
+ARG AIRFLOW_HOME=/usr/local/airflow
+
+# Define en_US.
+ENV LANGUAGE en_US.UTF-8
```

Comment:
if you care you can combine multiple ENV calls together just to save layers

Reply:
saw the top comment, never mind :-D

Reply:
Done.